### PR TITLE
154027362 - Upgrade 'less-rails' and 'sprockets' gems

### PR DESCRIPTION
**Pull request by nisevi**
https://www.pivotaltracker.com/story/show/154027362

### CODE REVIEW

File: features/step_definitions/map_steps.rb [hunk 1]
```diff
@@ -104,9 +104,9 @@ def find_map_icon klass, org_id
   org_id = klass.find_by(name: name).id
   marker_class = (icon_size == "small") ? "measle" : "marker"
   if marker_class == "measle"
-    expect(find_map_icon(marker_class, org_id)["src"]).to end_with "/assets/measle.png"
+    expect(find_map_icon(marker_class, org_id)["src"]).to match /\/assets\/measle-(\w*)\.png/ix
   else
-    expect(find_map_icon(marker_class, org_id)["src"]).to end_with "/assets/marker.png"
+    expect(find_map_icon(marker_class, org_id)["src"]).to match /\/assets\/marker-(\w*)\.png/ix
```
Comment (tansaku) [inline features/step_definitions/map_steps.rb]: looking good - to replicate exactly should we include the `$` marker in the regex to indicate we expect this text to appear at the end of the src attribute as we did in the original test ...?

**Comment by tansaku** [comment]
great work here @nisevi - see my small note

File: features/step_definitions/map_steps.rb (hunk 1 above)
Comment (nisevi) [reply features/step_definitions/map_steps.rb]: @tansaku yes! I think you are right :+1: I will make the corresponding changes.

**Comment by nisevi** [comment]
@tansaku I upgraded the regex based on your suggestion, I think now matches what the original step was doing :+1:

**Pull request merged**